src/models/thelio-mega-r4-n3/repairs.md

```diff
@@ -0,0 +1,700 @@
+    - As visible in the photo below, the elongated screw holes allow the GPU brace to be slightly rotated, which can allow the GPU finger(s) to better support the exact position of the GPU.
+
+![GPU brace screws](./img/gpu-brace-screws.webp)
+
+3. Pull the GPU brace out of the chassis.
+
+### Steps to adjust the GPU fingers:
+
+1. Unscrew the four screws (two on each side) holding the GPU finger onto the GPU brace, highlighted cyan below.
+
+![GPU finger screws](./img/gpu-finger-screws.webp)
+
+2. Move the GPU finger to the desired position, then re-attach the screws.
+3. Repeat for any additional GPU fingers present.
+4. To adjust the depth the GPU brace extends into the chassis, loosen the screws highlighted yellow above, then tighten them when the GPU brace is in the desired position.
+
+## Removing the CPU duct:
+
+The CPU duct guides airflow through the CPU cooler. It covers the CPU and RAM slots on the motherboard.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 9 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to remove the CPU duct:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the side fan mount](#removing-the-side-fan-mount), and [remove the GPU brace](#removing-the-gpu-brace).
+2. Unscrew the four back screws holding the CPU duct in place.
+
+![CPU duct screws](./img/cpu-duct-screws-back.webp)
+
+3. Unplug the braided cables for the two CPU duct-mounted fans, highlighted light blue below.
+    - The non-braided Y-cable does not need to be unplugged at this stage.
+
+![Top exhaust fans](./img/cpu-duct-screws-connectors-top.webp)
+
+4. While holding the CPU duct up, unscrew the two top screws connecting it to the top crossbar, highlighted green above.
+5. Pull the CPU duct out of the machine.
+
+## Replacing a GPU:
+
+Thelio Mega R4-N3 contains four PCI Express 5.0 x16 slots. Factory configurations are offered with up to three GPUs. Four GPUs may function, depending on space and power constraints.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 15 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to replace a GPU:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the side fan mount](#removing-the-side-fan-mount), and [remove the GPU brace](#removing-the-gpu-brace).
+2. If you're removing a GPU, unplug the GPU power cable from the right side of the card. Hold down the latch on the connector while unplugging the cable.
+
+![GPU power connections](./img/gpu-power.webp)
+
+3. Loosen the two back screws holding the PCIe bracket in place, then slide the PCIe bracket open.
+    - You can optionally remove the screws and PCIe bracket instead.
+
+![PCIe bracket screws](./img/pcie-bracket.webp)
+
+4. Press the latch on the motherboard to free the PCIe connection, then pull the card out of the slot.
```

Comment:
The top slot has a latch release button further to the right on the board, just below the big heatsink with the silver grille. Mentioning this might also negate the need to mention removing the bottom RAM stick in the line below this one?

Reply:
I actually pushed that button a few times trying to figure out what it did and didn't see anything, lol. I'm going to double-check if it's even working on our board (the latch on the photo unit's PCIe slot is partially broken off, so not sure if something wonky's going on with it), but it's definitely worth at least mentioning as a potential option.

Reply:
Switched to recommending that button for the top slot, removed reference to taking the bottom RAM stick out. (This button still requires removing the CPU duct to reach.)